### ISSUE TITLE
add interaction types  ectomycorrhizal (EM) and arbuscular mycorrhizal (AM) 

### ISSUE THREAD

**Issue by jhpoelen** [oborel/obo-relations#325]
Suggest to add interaction types  ectomycorrhizal (EM) and arbuscular mycorrhizal (AM) . See https://github.com/globalbioticinteractions/mycodb/issues/2 and Chaudhary, V. B., Rúa, M. A., Antoninka, A., Bever, J. D., Cannon, J., Craig, A., … Hoeksema, J. D. (2016). MycoDB, a global database of plant response to mycorrhizal fungi. Scientific Data, 3, 160028. [doi:10.1038/sdata.2016.28](https://doi.org/10.1038/sdata.2016.28) .

**Comment by cmungall** [comment]
What would the labels be?

E.g.

  * has host
    * has host, root associated
       * has host, ectomycorrhizal
       * has host, arbuscular mycorrhizal

we would have swrl rules that would axiomatize these in terms of PO classes like root cortex

cc @diatomsRcool

**Comment by jhpoelen** [comment]
Looks good to me!

**Comment by diatomsRcool** [comment]
looks good to me too

**Comment by mellybelly** [comment]
swrl rules need both PO terms and taxon, yes? because they wouldn't necessarily apply to all taxa that the PO term applies to?

**Comment by cmungall** [comment]
Yes, taxon to be able to infer it's fungal (the plant part is implicit from
usage of PO)

**Comment by nlharris** [comment]
It seems like there was agreement here about what's needed, and someone needs to make the change in a PR?

**Comment by jhpoelen** [comment]
@nlharris suggest to re-open this issue. 

As far as I know, the proposed biotic interactions are quite common in soil ecology and botany (ectomycorrhizal (EM) and arbuscular mycorrhizal (AM)). The RO process and my personal availability and skills to add interaction types is such that it takes years to add an interaction type to RO. 

I much appreciate RO and their curators, and I hope you'll considering re-opening this issue.

**Comment by jhpoelen** [comment]
@balhoff thanks!

**Comment by jhpoelen** [comment]
I would very much like to discuss the ectomycorrhizal (EM) and arbuscular mycorrhizal (AM) interaction types in future meetings.

@cmungall @diatomsRcool any ideas on how to restart the introduction of the EM and AM interactions, especially now that https://globalbioticinteractions.org/microbenetnet has kicked off in March 2025!

**Comment by jhpoelen** [comment]
Thanks @balhoff  . 

@diatomsRcool @cmungall are you still interested to pursue these neat mycorrhizal interaction types? Would be nice to have a little collaboration instead of going at it alone. . .

**Comment by diatomsRcool** [comment]
sure! what do you need from me?

**Comment by jhpoelen** [comment]
@diatomsRcool thanks for responding. 

@cmungall suggested to 

> we would have swrl rules that would axiomatize these in terms of PO classes like root cortex

and I am not quite sure how to address that. How would you approach introducing arbuscular / ecto mycorrhizal interaction types with these rule?

**Comment by jhpoelen** [comment]
Terms have been added. Thank you @cmungall @balhoff @diatomsRcool @mellybelly @nlharris  for helping making this happen! 

Examples are - 
http://purl.obolibrary.org/obo/RO_0002807
http://purl.obolibrary.org/obo/RO_0002805


With usage seen on 2026-07-08 - 

https://www.globalbioticinteractions.org/?interactionType=hasArbuscularMycorrhizalHost


<img width="1440" height="742" alt="Image" src="https://github.com/user-attachments/assets/1255e07d-6398-4dbb-b4e3-70df855d439a" />
